File: mpp/Lab7/javaPb9/networking/src/main/java/ro/mpp2024/DTO/RezultatDTO.java
package ro.mpp2024.DTO;

import java.io.Serializable;

public class RezultatDTO implements Serializable {
    private String idProba;
    private Long idParticipant;
    private String numeParticipant;
    private String prenumeParticipant;
    private long puncte;

    public RezultatDTO() {
    }

    public RezultatDTO(String idProba, String numeParticipant, String prenumeParticipant, Long idParticipant, long puncte) {
        this.idProba = idProba;
        this.numeParticipant = numeParticipant;
        this.prenumeParticipant = prenumeParticipant;
        this.idParticipant = idParticipant;
        this.puncte = puncte;
    }

    public String getIdProba() {
        return idProba;
    }

    public Long getIdParticipant() {
        return idParticipant;
    }

    public long getPuncte() {
        return puncte;
    }

    public String getNumeParticipant() {
        return numeParticipant;
    }

    public String getPrenumeParticipant() {
        return prenumeParticipant;
    }

    @Override
    public String toString() {
        return "RezultatDTO{" +
                "idProba='" + idProba + '\'' +
                ", idParticipant=" + idParticipant +
                ", puncte=" + puncte +
                '}';
    }
}

File: mpp/Lab12/csharpPb9/networking/DTO/RezultatDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace networking.DTO
{
    [Serializable]
    internal class RezultatDTO
    {
        [JsonPropertyName("idProba")]
        public String idProba { get; set; }
        [JsonPropertyName("idParticipant")]
        public long idParticipant { get; set; }
        [JsonPropertyName("numeParticipant")]
        public String numeParticipant { get;  set; }
        [JsonPropertyName("prenumeParticipant")]
        public String prenumeParticipant { get;  set; }
        [JsonPropertyName("puncte")]
        public long puncte { get; set; }


        public RezultatDTO() { }

        public RezultatDTO(String idProba, long idParticipant, String numeParticipant, String prenumeParticipant, long puncte)
        {
            this.idProba = idProba;
            this.idParticipant = idParticipant;
            this.numeParticipant = numeParticipant;
            this.prenumeParticipant = prenumeParticipant;
            this.puncte = puncte;
        }

        public override string ToString()
        {
            return $"RezultatDTO [idProba={idProba}, idParticipant={idParticipant}, numeParticipant={numeParticipant}" +
                $", prenumeParticipant={prenumeParticipant}, puncte={puncte}]";
        }
    }
}
